.github/workflows/ci_tests_dev.yaml

```
@@ -23,7 +23,7 @@ on:
     - cron: '0 0 * * *'
 
 jobs:
-  test_gmt_master:
+  test_gmt_main:
```

It means "test the GMT master branch", so it should stay with "master", but I think it's OK to change it to "test_gmt_dev:
```suggestion
  test_gmt_dev:
```